package.json

```diff
@@ -10,8 +10,7 @@
     "ts-node": "ts-node -T -P tsconfig.node.json",
     "courses-gen": "npm run ts-node -- src/requirements/courses-json-generator.ts",
     "req-gen": "npm run ts-node -- src/requirements/requirement-json-generator.ts",
-    "req-graph-dump": "npm run ts-node -- src/requirements/admin/requirement-graph-dumper.ts",
-    "track-users": "npm run ts-node -- src/requirements/admin/track-users.ts",
+    "track-users": "npm run ts-node -- scripts/track-user-analytics.ts",
```

Comment:
as long as this works (we can run it with npm), then I fully expect the track-users workflow to continue working, so we should be good.

Reply:
LOL whoops. I just tested it and it broke since I actually renamed it back to `track-users.ts`. Should work now